StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentMethodCollectionView.swift

```diff
@@ -208,15 +243,32 @@ extension SavedPaymentMethodCollectionView {
             }
         }
 
-        // MARK: - Internal Methods
+        private lazy var labelBottomConstraint: NSLayoutConstraint = {
+            return label.bottomAnchor.constraint(equalTo: contentView.bottomAnchor)
+        }()
+        private lazy var labelHeightConstraint: NSLayoutConstraint = {
+            return label.heightAnchor.constraint(equalToConstant: 20)
+        }()
+        private lazy var defaultBadgeConstraints: [NSLayoutConstraint] = {
+            return [
+                defaultBadge.topAnchor.constraint(
+                    equalTo: label.bottomAnchor, constant: 4),
+                defaultBadge.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+                defaultBadge.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 2),
+                defaultBadge.trailingAnchor.constraint(equalTo: contentView.trailingAnchor)
+            ]
+        }()
 
-        func setViewModel(_ viewModel: SavedPaymentOptionsViewController.Selection, cbcEligible: Bool, allowsPaymentMethodRemoval: Bool) {
+        // MARK: - Internal Methods
+        func setViewModel(_ viewModel: SavedPaymentOptionsViewController.Selection, cbcEligible: Bool, allowsPaymentMethodRemoval: Bool, allowsSetAsDefaultPM: Bool = false, showDefaultPMBadge: Bool = false) {
```

Comment:
Is there a way to make `allowsSetAsDefaultPM`, `showDefaultPMBadge` and `isDefaultPM` less confusing?  I understand isDefaultPM actually shows the badge, but I'm getting confused between `allowsSetAsDefaultPM`, and `showDefaultPMBadge`.

Reply:
I've changed it to allowsSetAsDefaultPM, needsVerticalPaddingForBadge, and showDefaultPMBadge. 

allowsSetAsDefaultPM is for the purpose of displaying the circular edit icon when "edit" is pressed— if a user is allowed to set a default payment method, if saved payment methods can be removed, or if a card is cbc eligible, then the user should have the option of managing their payment method and navigating to the update payment method screen.

needsVerticalPaddingForBadge indicates that there exists a set default payment method, and therefore, the cells need padding on the bottom to accommodate for the default badge. The default badge may not be shown on this cell, but it needs the padding regardless to match the cell that has the badge.

showDefaultPMBadge, now, is only true for the cell with the default payment method. When this is true, the default badge is displayed.